Assets/Modules/AdvancedPassword/AdvancedPassword.cs

```diff
@@ -422,6 +429,16 @@ void Update()
                         yield return "sendtochaterror Unknown dial: " + command[0];
                         yield break;
                     }
+                    if (!int.TryParse(command[2], out temp))
+                    {
+                        yield return "sendtochaterror Invalid number: " + command[2];
+                        yield break;
+                    }
+                    if (temp < 0)
```

Comment:
Negative numbers should be accepted, -1 is perfectly reasonable as a replacement for 11.